app.js

```diff
@@ -1,5 +1,8 @@
 /* application specific logic */
 
+// babel-polyfill is needed to modify existing builtins to support newer syntax
```

Comment:
I don't find that source code comment useful. That's what babel-polyfill does so whoever sees `import "babel-polyfill";` has that information already. What I think this comment should say is something like "required by at least IE11".

And is the comment really accurate? Syntax cannot be really handled by a polyfill, only a transpiler can deal with new syntax.